### PR TITLE
ref: improve rwd of registration layout #P23-198

### PR DESCRIPTION
**Pull request by AzulloCyll**

### CODE REVIEW

File: apps/web/app/(regflow)/sign-in/layout.styles.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    flex-direction: column;
+    justify-content: center;
+    padding-left: 6%;
+    padding-right: 3%;
+  }
+`;
+
+export const FormWrapperStyled = styled.div`
+  overflow-x: auto;
+  padding: 8px;
```
Comment (wyola) [inline apps/web/app/(regflow)/sign-in/layout.styles.tsx:48]: padding is very small, especially on the right side, while there is a lot of space between h1 and the form wrapper

image

Comment (AzulloCyll) [reply apps/web/app/(regflow)/sign-in/layout.styles.tsx]: Thanks for your review. Changed this padding to 16px. But many other changes was added including better example of sign-up page.

File: apps/web/app/(regflow)/layout.styles.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+  ${device.desktop} {
+    margin-top: 32px;
+    font-size: 2.5rem;
+    line-height: 2.5rem;
+  }
+`;
+
+export const ParagraphStyled = styled.p`
+  font-family: "Inter", sans-serif;
```
Comment (aniasgit) [inline apps/web/app/(regflow)/layout.styles.tsx:71]: We have set Inter as the default font in our RootLayout, so there is no need to add font-family here. Especially that in this form it doesn't work and Arial is rendered (for me) and not Inter.

File: apps/web/app/(regflow)/layout.styles.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+
+  ${device.desktop} {
+    font-size: 1rem;
+    line-height: 1.5rem;
+    margin-top: 1rem;
+  }
+`;
+
+export const CustomCard = styled(Card)`
+  font-family: "Inter", sans-serif;
```
Comment (aniasgit) [inline apps/web/app/(regflow)/layout.styles.tsx:85]: same as here: https://github.com/intive/patronage2023-js/pull/29/files#r1155777752

**Review by wyola — COMMENTED** [review]
In this view, I would make the card the same width as the h1, but it is just my personal preference and I don't know what was decided about it during the layout call

image

File: apps/web/app/(regflow)/layout.styles.tsx (hunk 2 above)
Comment (aniasgit) [reply apps/web/app/(regflow)/layout.styles.tsx]: after removing font-family, inter is render now :)

File: apps/web/lib/media-queries.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+const size = {
+  desktop: '768px',
+  large: '1200px'
+};
+
+export const device = {
+  desktop: `@media (min-width: ${size.desktop})`,
+  large: `@media (min-width: ${size.large})`,
+};
```
Comment (wyola) [inline apps/web/lib/media-queries.tsx:9]: add empty line at the end

**Comment by monika-klokosinska-intive** [comment]
Screenshot 2023-04-04 at 18 06 12
@AzulloCyll I am getting a weird behaviour around 1020px width - a horizontal scroll or a cut down view. Could you take a look?

**Comment by monika-klokosinska-intive** [comment]
@AzulloCyll nevermind, I am not able to reproduce. Let's chalk it up to my local's weirdness and leave it as it!

**Review by aniasgit — APPROVED** [review]
It looks good to me 👍

**Review by monika-klokosinska-intive — APPROVED** [review]

**Pull request merged**